### DISCUSSION TITLE
MSC in english and Abstract in portuguese first

### DISCUSSION THREAD

**Discussion by amasilva** [joaomlourenco/novathesis#406] [Q&A] [answered]
Hi,
I would like to know if i can write the thesis in English as main language but change the order of the abstracts.
Thank you

**Comment by joaomlourenco** [comment, marked as answer]
Yes you can! 😃
But beware that if you are doing your MSc or PhD at FCT-NOVA, you **should not** change that!

In file `Config/5_packages.tex` add
```latex
\abstractorder(en):={pt,en}
```
